### ISSUE TITLE
 moderate severity vulnerabilities from package minimist

### ISSUE THREAD

**Issue by nfathan** [symfony/webpack-encore#711]
got warning about vulnerability when execute `npm audit`

[Prototype Pollution - official npm advisory](https://npmjs.com/advisories/1179)

```bash
                            Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > assets-webpack-plugin > mkdirp >    
                  minimist                                                      
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > babel-loader > mkdirp > minimist    
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > optimize-css-assets-webpack-plugin  
                  > cssnano > cssnano-preset-default > postcss-svgo > svgo >    
                  mkdirp > minimist                                             
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > terser-webpack-plugin > cacache >   
                  mkdirp > minimist                                             
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > terser-webpack-plugin >   
                  cacache > mkdirp > minimist                                   
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > terser-webpack-plugin > cacache >   
                  move-concurrently > copy-concurrently > mkdirp > minimist     
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > terser-webpack-plugin >   
                  cacache > move-concurrently > copy-concurrently > mkdirp >    
                  minimist                                                      
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > terser-webpack-plugin > cacache >   
                  move-concurrently > mkdirp > minimist                         
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > terser-webpack-plugin >   
                  cacache > move-concurrently > mkdirp > minimist               
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > mkdirp > minimist         
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack-dev-server > portfinder >   
                  mkdirp > minimist                                             
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack-dev-server >                
                  webpack-dev-middleware > mkdirp > minimist                    
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > watchpack > chokidar >    
                  fsevents > node-pre-gyp > mkdirp > minimist                   
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack-dev-server > chokidar >     
                  fsevents > node-pre-gyp > mkdirp > minimist                   
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > watchpack > chokidar >    
                  fsevents > node-pre-gyp > tar > mkdirp > minimist             
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack-dev-server > chokidar >     
                  fsevents > node-pre-gyp > tar > mkdirp > minimist             
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack > watchpack > chokidar >    
                  fsevents > node-pre-gyp > rc > minimist                       
                                                                                
  More info       https://npmjs.com/advisories/1179                             
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         minimist                                                      
                                                                                
  Patched in      >=1.2.3                                                       
                                                                                
  Dependency of   @symfony/webpack-encore [dev]                                 
                                                                                
  Path            @symfony/webpack-encore > webpack-dev-server > chokidar >     
                  fsevents > node-pre-gyp > rc > minimist                       
                                                                                
  More info       https://npmjs.com/advisories/1179   
                                  
found 18 moderate severity vulnerabilities in 12017 scanned packages
  18 vulnerabilities require manual review. See the full report for details.
```

**Comment by stof** [comment]
none of them are direct dependencies of webpack-encore. This should be reported to projects depending on the old version of `mkdirp`.

**Comment by stof** [comment]
And this has now been solved by mkdirp with their 0.5.2 and 0.5.3 releases. Run `yarn upgrade` and you will be good.
